### PR TITLE
Add information about lineage permissions to Astro

### PR DESCRIPTION
**Pull request by jwitz**
Resolves https://github.com/astronomer/docs/issues/873

### CODE REVIEW

File: astro/data-lineage.md [hunk 1]
```diff
@@ -19,7 +19,13 @@ From the **Lineage** tab on Astro, you can access the following four pages:
 
 You can use these pages to diagnose issues that may be difficult to troubleshoot in other environments. For example, if an Airflow task failed because a database schema changed, you can use the **Lineage** page on Astro to determine which job caused the change and which downstream tasks failed as a result.
 
-All members of your Organization can view the **Lineage** page. For more information on data lineage and related concepts, see [Data lineage concepts](data-lineage-concepts.md).
+For more information on data lineage and related concepts, see [Data lineage concepts](data-lineage-concepts.md).
+
+:::caution
+
+All members of your Organization can view the **Lineage** tab regardless of their Workspace permissions. The **Lineage** tab can contain plain-text SQL and Python code from any system that you configure to emit lineage data to Astro. If this is a security concern for your organization, reach out to [Astronomer support](https://support.astronomer.io/).
```
Comment (jwitz) [inline astro/data-lineage.md:26]:
```suggestion
All members of your Organization can view the **Lineage** tab regardless of their Workspace permissions. The **Lineage** tab can contain plain-text SQL and Python code from any system that emits lineage data to Astro. If this is a security concern for your organization, reach out to [Astronomer support](https://support.astronomer.io/).
```

File: astro/data-lineage.md [hunk 2]
```diff
@@ -19,7 +19,13 @@ From the **Lineage** tab on Astro, you can access the following four pages:
 
 You can use these pages to diagnose issues that may be difficult to troubleshoot in other environments. For example, if an Airflow task failed because a database schema changed, you can use the **Lineage** page on Astro to determine which job caused the change and which downstream tasks failed as a result.
 
-All members of your Organization can view the **Lineage** page. For more information on data lineage and related concepts, see [Data lineage concepts](data-lineage-concepts.md).
+For more information on data lineage and related concepts, see [Data lineage concepts](data-lineage-concepts.md).
+
+:::caution
+
+All members of your Organization can view the **Lineage** tab regardless of their Workspace permissions. The **Lineage** tab can contain plain-text SQL and Python code from any system that emits lineage data to Astro. If this is a security concern for your organization, reach out to [Astronomer support](https://support.astronomer.io/).
```
Comment (paolaperaza) [inline astro/data-lineage.md:26]:
```suggestion
All members of your Astro Organization can view the **Lineage** tab regardless of their Workspace permissions. The **Lineage** tab could contain plain-text SQL and Python code from any system that emits lineage data to Astro. If this is a security concern for your organization, reach out to [Astronomer support](https://support.astronomer.io/).
```

File: astro/user-permissions.md [hunk 3]
```diff
@@ -13,13 +13,14 @@ To better protect your data pipelines and cloud infrastructure, Astro offers rol
...
-| Invite new user to Organization       |                       |                              | ✔️                     |
+| Permission                                           | **Organization Member** | **Organization Billing Admin** | **Organization Owner** |
+| ---------------------------------------------------- | ----------------------- | ------------------------------ | ---------------------- |
+| Create a new Workspace                               | ✔️                      | ✔️                             | ✔️                     |
+| View Organization details and user membership        | ✔️                      | ✔️                             | ✔️                     |
+| View lineage data in the **Lineage** tab             | ✔️                      | ✔️                             | ✔️                     |
+| Update Organization billing information and settings |                         | ✔️                             | ✔️                     |
+| Update Organization user membership                  |                         |                                | ✔️                     |
```
Comment (paolaperaza) [inline astro/user-permissions.md:22]:
```suggestion
| Update roles and permissions of existing Organization users                  |                         |                                | ✔️                     |
```

This sounds more clear to me?

**Review by paolaperaza — APPROVED** [review]
Minor comments here @jwitz but LGTM ! Thanks for this, and apologies for delay 🙏

**Pull request merged**